crates/partition-store/src/partition_store_manager.rs

```diff
@@ -80,24 +90,25 @@ impl PartitionStoreManager {
 
         Ok(Self {
             rocksdb,
-            lookup: Arc::default(),
+            partition_stores: Arc::default(),
+            persisted_lsns,
         })
     }
 
     /// Check whether we have a partition store for the given partition id, irrespective of whether
     /// the store is open or not.
     pub async fn has_partition_store(&self, partition_id: PartitionId) -> bool {
-        let _guard = self.lookup.lock().await;
+        let _guard = self.partition_stores.lock().await;
```

Comment:
not really sure if that rename adds anything other than introducing irrelevant changes to the PR

Reply:
reverted; I thought it was marginally useful to differentiate what the field is for, given we now have more than one lookup map in the struct.